eng/Packages.Data.props

```diff
@@ -138,7 +138,7 @@
     <PackageReference Update="System.Text.Json" Version="4.6.0" />
 
     <!-- Build time packages -->
-    <PackageReference Update="Azure.ClientSdk.Analyzers" Version="0.1.1-dev.20200721.2" />
+    <PackageReference Update="Azure.ClientSdk.Analyzers" Version="0.1.1-dev.20200929.2" />
```

Comment:
cc @annelo-msft

Reply:
@pakrym is this and fyi for @annelo-msft or do you have any concerns about this? I wonder if I can go ahead with merging or there are things that need to be verified first.

Reply:
Sorry, I thought I approved. No concerns, I just remembered that Anne needed this updated at one point.

Reply:
Thanks for the FYI @pakrym !